web/core/components/issues/issue-layouts/filters/header/filters/additional-properties.tsx

```
@@ -88,9 +88,7 @@
                 <button
                   onClick={() => fetchNextPage()}
                   className="ml-8 text-xs font-medium text-custom-primary-100 cursor-pointer"
-                >
-                  
-                </button>
+                 />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)